cmd/pebble/main.go

```diff
@@ -93,9 +94,10 @@ func main() {
 		chainLength = int(val)
 	}
 
-	db := db.NewMemoryStore()
-	ca := ca.New(logger, db, c.Pebble.OCSPResponderURL, alternateRoots, chainLength, c.Pebble.CertificateValidityPeriod)
-	va := va.New(logger, c.Pebble.HTTPPort, c.Pebble.TLSPort, *strictMode, *resolverAddress)
+	clockSource := clock.New()
```

Comment:
Based on your PR description (you mention "breaking the external API") it sounds like you're using Pebble as a library, rather than as a standalone binary? In that case, your changes here make sense.

But if we're going to go to the trouble of giving Pebble full fake clock support, we should give it to everyone, including people who use Pebble as a binary (which I think is _most_ users of Pebble).

To that end, let's replace this line with something a bit more like [this](https://github.com/letsencrypt/boulder/blob/main/cmd/clock_integration.go), so that the fake clock can be initialized from an environment variable if someone wants to, or defaults to the system clock otherwise.